content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,186 @@
+============
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard, which can be found by going to :menuselection:`Payroll
+--> Work Entries --> Work Entries`, provides a visual overview of the individual time sheets of
+every employee, with each day split into a morning shift and an afternoon shift.
+
+.. image:: work_entries/work-entries-overview.png
+   :align: center
+   :alt: Work Entries dashboard view showing all employee's work entries.
+
+To change the view so that only the entries for a single day, week, or month are shown, click on
+either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month`.
+
+.. image:: work_entries/work-entry-date.png
+   :align: center
+   :alt: Select the date format to view, either day, week, or month.
+
+Use the :guilabel:`arrow icons` on the left and right side of the :guilabel:`Today` button to adjust
+the displayed dates. The arrows will adjust the date based on the type of time selected. For
+example, if month is selected, the arrows will move one month with each click of the arrow. If week
+or day is selected, the time will move by either a week or a day for each click of the arrow,
+respectively.
+
+Adding a new work entry
+=======================
+
+If a work entry is missing and needs to be added, such as  sick time or time off, click
+:guilabel:`Add` to create a new work entry. A pop-up will appear, with several fields to fill in.
+
+Enter the :guilabel:`Work Entry Name`, such as 'Sick Time' or any other short description. Select
+the :guilabel:`Employee` and the :guilabel:`Work Entry Type` from the respective drop-down lists.
+
+.. image:: work_entries/create.png
+   :align: center
+   :alt: Create a new work entry by clicking Create and entering the information.
+
+Next, enter the date and time for the work entry in the :guilabel:`From` and :guilabel:`To`
+drop-downs. First, select the date by navigating to the correct month and year using the
+:guilabel:`arrow icons` icons, then click on the specific day.
+
+.. image:: work_entries/date.png
+   :align: center
+   :alt: Select the date from the calendar using the drop down arrow.
+
+Then, select the time by clicking on the :guilabel:`clock icon`, and using the :guilabel:`up and
+down arrows` for each section to enter the hour, minute, and second for the time period.
+
+.. image:: work_entries/time.png
+   :align: center
+   :alt: Enter the time in hours, minutes, and seconds, using the arrows.
```

Comment:
Don't think you need either screenshot here since this is a common feature across Odoo. Imo it's enough to just say what to do in written form as long as it's clear 👍 

As well, would fix at the gui + emoji formatting here (similar to above comment I made earlier).
The formatting for those is: 
```
:guilabel:`emoji (icon name)`
```